### PR TITLE
[Needs Review] B07902123 蔡奇夆

### PR DESCRIPTION
**Pull request by ktpss95112**
A Pull Request (PR) should explain why we need this PR and how you implement it, so others could understand it quickly.
Since this is just a homework, you do not need to explain why and how. 

**Do NOT change the status to NeedsReview if you do not want to be graded yet.**

Please fill in the following questions as feedback. It is totally optional and would not be graded.

## Is this homework helpful for you?
Yes
## Is there any suggestion?

<br />

### CODE REVIEW

File: src/students/b07902123.py [hunk 1]
```diff
@@ -0,0 +1,427 @@
...
+    def draw_text(
+        img: Image,
+        text: str,
+        location: tuple = (0, 0),
+        text_color=(0, 0, 0)
+    ) -> Image:
+        draw = ImageDraw.Draw(img)
+        font = ImageFont.truetype("DejaVuSans.ttf", 20)
+        draw.text(location, text, font=font, fill=text_color)
+        return img
```
Comment (amjltc295) [inline src/students/b07902123.py]: draw_text() is already defined in `utils.py`. Don't define it again.

Comment (ktpss95112) [reply src/students/b07902123.py]: Updated.
Please review it again.

Comment (amjltc295) [reply src/students/b07902123.py]: I can't see any update.

**Comment by amjltc295** [comment]
LGTM

**Pull request merged**